src/bench/rpc_blockchain.cpp

```diff
@@ -7,12 +7,15 @@
 
 #include <rpc/blockchain.h>
 #include <streams.h>
+#include <test/util/setup_common.h>
 #include <validation.h>
 
 #include <univalue.h>
 
 static void BlockToJsonVerbose(benchmark::Bench& bench)
 {
+    TestingSetup test_setup{};
```

Comment:
Doesn't matter here, but it would be good to specify `-nodebuglogfile` to be on the safe side

Reply:
Fixed in #21003